.gitmodules

```diff
@@ -1,23 +1,23 @@
 [submodule "seastar"]
 	path = seastar
-	url = ../seastar
+	url = ../../scylladb/seastar
```

Comment:
The existing path is intentional. If we were to move a provider or rename the organization, then a .gitmodules file that names the organization would point at the wrong place.

Cloning/updating the submodules is annoying, but a script can do the job.

Reply:
Although I think `.gitmodule` is the least concern if changing a provider or renaming the organization, and although we don't have a convenient script to handle this burden, I am dropping this PR - it is not THAT important.